lib/cc/analyzer/container.rb

```diff
@@ -60,7 +60,7 @@ def run(options = [])
           duration = timeout
           @listener.timed_out(container_data(duration: duration))
         else
-          sleep 0.01 until !t_out.alive? && !t_err.alive?
+          sleep 0.2 until !t_out.alive? && !t_err.alive?
```

Comment:
I wondered this last time it came though: doesn't this basically just re-implement `Thread#join`?